torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -9,9 +9,9 @@
 #include <torch/csrc/jit/tensorexpr/lowerings.h>
 #include <torch/csrc/jit/tensorexpr/tensor.h>
 
-namespace torch {
-namespace jit {
-namespace tensorexpr {
+#include <utility>
```

Comment:
One again, put includes at top not in the nested scope.

Reply:
Fixed